components/About/CallToAction.tsx

```diff
@@ -60,30 +69,31 @@ export const CallToAction = ({
               ))}
             </ul>
           )}
-          {earn > 0 && !comingSoon && (
+          {earn > 0 && !comingSoon && !ended && (
             <div className="bg-ifpink px-4 py-2 inline-block mt-2 text-xs md:text-md">
               Earn up to {earn.toLocaleString('en-US')} points a week
             </div>
           )}
           <div className="mb-8" />
-          {ctaText && href && button ? (
-            <Link href={href}>{button}</Link>
-          ) : (
-            button
-          )}
-          {submissionForm && (
-            <Link href="https://forms.gle/yrAtzoyKTwLgLTRZA" passHref>
-              <RawButton
-                border="border"
-                className="m-auto w-full mt-2 max-w-md mb-2 text-md p-2"
-                colorClassName="text-black bg-transparent hover:bg-black hover:text-white"
-              >
-                <Link href={'https://forms.gle/yrAtzoyKTwLgLTRZA'}>
-                  Claim Points
-                </Link>
-              </RawButton>
-            </Link>
-          )}
+          {!ended &&
+            (ctaText && href && button ? (
+              <Link href={href}>{button}</Link>
+            ) : (
+              button
+            )) &&
+            submissionForm && (
```

Comment:
This logic looks like it changed?

I believe it should be `!ended && submissionForm` and not combined with the previous conditional button.